README.md

```diff
@@ -13,7 +13,7 @@ Role Variables
 
 Defaults: `defaults/main.yml`
 
-- `postgresql_version`: The PostgreSQL major version: `9.4` (default), `9.5`, `9.6` or `10`
+- `postgresql_version`: The PostgreSQL major version: `9.4` (default), `9.5`, `9.6`, `10`, `11`
```

Comment:
I assume changing the default would be a breaking change.

Reply:
I initially thought about that and reverted the change since it will be breaking

Reply:
To reduce the testing matrix I think this addition of PG11 is a good time to drop 9.4 (and 9.5 too?).

Reply:
And remove the default altogether and force listing a version?

Reply:
I think making a the version mandatory is a good idea.